src/dashboard.ts

```diff
@@ -28,20 +28,29 @@ export async function dashResults(
   db: Database
 ): Promise<{ timeSeries: Record<string, number[]> }> {
   const result = await db.query(
-    `SELECT rank_filter.* FROM (
-      SELECT t.startTime, m.iteration, value, b.name as benchmark,
-        rank() OVER (
-          PARTITION BY b.id
-          ORDER BY t.startTime DESC, m.iteration DESC
-        )
-        FROM Measurement m
-          JOIN Trial t ON  m.trialId = t.id
-          JOIN Experiment e ON t.expId = e.id
-          JOIN Run r ON m.runId = r.id
-          JOIN Benchmark b ON r.benchmarkId = b.id
-        WHERE projectId = $1
-        ORDER BY t.startTime, m.iteration
-      ) rank_filter WHERE RANK <= 100`,
+    `WITH explodedValueMeasurement AS(
+      WITH orderedMeasurement AS(
+        SELECT runid,trialid,criterion,invocation,value 
+        FROM measurement ORDER BY runid,trialid,criterion,invocation ASC)
+      SELECT m.runid,m.trialid,m.criterion,m.invocation,a.value, a.iteration 
+      FROM orderedMeasurement as m
+      LEFT   JOIN LATERAL unnest(m.value)
+                          WITH ORDINALITY AS a(value, iteration) ON true 
+      )
```

Comment:
Hm, this looks complicated.
Is this something we really want to do in the database?
I wonder what the tradeoff here is.

If this is faster in the database, perhaps we should use the same approach for the R scripts? I would somehow assume that this would mean that more data needs to be transferred.

Could we measure this?

Reply:
Could you please also format the code to be more consistent with the existing code style?
- whitespace indentation
- space between keywords
- space after `,`